apps/etterlatte-saksbehandling-ui/client/src/components/behandling/attestering/oppsummering/oversikt.tsx

```diff
@@ -28,8 +28,8 @@ export const Oversikt = ({
   const kommentarFraAttestant = attestering?.kommentar
   const oppgave = useSelectorOppgaveUnderBehandling()
 
-  const [res, hentNavnForIdent] = useApiCall(hentNavnforIdent)
-  const [attestant, hentNavnForAttestant] = useApiCall(hentNavnforIdent)
+  const [saksbehandlerNavn, hentNavnForIdent] = useApiCall(hentNavnforIdent)
+  const [attestantNavn, hentNavnForAttestant] = useApiCall(hentNavnforIdent)
```

Comment:
Hvorfor ha samme API kall-state 2 ganger?

Reply:
Eller er `hentNavnForIdent` forskjellige? Isåfall burde de ha forskjellige navn

Reply:
Nei de er ikke forskjellige men de hentes på to ulike måter her og får da ulike statuser osv